csrankings.csv

```diff
@@ -16969,6 +16975,7 @@ Yadati Narahari,IISc Bangalore,http://www.csa.iisc.ac.in/~hari,NOSCHOLARPAGE
 Yadong Mu,Peking University,http://www.muyadong.com,Fqqx4HsAAAAJ
 Yael Amsterdamer,Bar-Ilan University,https://u.cs.biu.ac.il/~amstery,70iHzEUAAAAJ
 Yafei Dai,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=5999,VSvWlvkAAAAJ
+Yafeng Yin,Nanjing University,http://cs.nju.edu.cn/yafeng/,v5CGgS8AAAAJ

```

Comment:
This person's web page says they are an "assistant researcher", which does not sound eligible for inclusion.